.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
```

Comment:
Vi må også legge til sånn at vi sjekker navpakkene: 
![image](https://github.com/bekk/nav-familie-endringsmelding-api/assets/17828446/0ee84b2c-9c77-45f8-874d-3df357eb1df6)

Se her for inspirasjon:
https://github.com/navikt/familie-ba-sak/blob/master/.github/dependabot.yml

Reply:
Secreten som skal legges under passord er `NPM_AUTH_TOKEN`

Reply:
Skal dette også legges til i dependabot for `nav-endringsmelding` repoet?